### PR TITLE
Bug 1291535 - alignment of oc commands in next steps page is wrong

### PR DESCRIPTION
**Pull request by jwforres**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1291535

### CODE REVIEW

**Comment by jwforres** [comment]
@spadgett PTAL

File: assets/app/views/create/nextSteps.html [hunk 1]
```diff
@@ -46,9 +46,9 @@ <h1 ng-if="!pendingTasks(tasks()).length && erroredTasks(tasks()).length">Comple
         <p>You may want to use the <code>oc</code> command line tool to help with troubleshooting. After <a target="_blank" href="{{'get_started_cli' | helpLink}}">downloading and installing</a> it, you can log in, switch to this particular project, and try some commands :</p>
 
         <pre class="code prettyprint">oc login {{loginBaseUrl}}
-  oc project {{projectName}}
-  oc build-logs -h
-  oc logs -h</pre>
+oc project {{projectName}}
+oc build-logs -h
+oc logs -h</pre>
```
Comment (spadgett) [inline assets/app/views/create/nextSteps.html]: This is fine, but wondering if we should make this a `div` with `white-space: pre-line;` instead. Then we don't have to be so careful with leading space.

File: assets/app/views/create/nextSteps.html [hunk 2]
```diff
@@ -46,9 +46,9 @@ <h1 ng-if="!pendingTasks(tasks()).length && erroredTasks(tasks()).length">Comple
         <p>You may want to use the <code>oc</code> command line tool to help with troubleshooting. After <a target="_blank" href="{{'get_started_cli' | helpLink}}">downloading and installing</a> it, you can log in, switch to this particular project, and try some commands :</p>
 
         <pre class="code prettyprint">oc login {{loginBaseUrl}}
-  oc project {{projectName}}
-  oc build-logs -h
-  oc logs -h</pre>
+oc project {{projectName}}
+oc build-logs -h
```
Comment (kargakis) [inline assets/app/views/create/nextSteps.html]: We have deprecated `build-logs` so I think it makes sense to not include it anywhere anymore

Comment (jwforres) [inline assets/app/views/create/nextSteps.html]: thanks for the heads up, i'll update this

**Comment by jwforres** [comment]
removed build-logs command

@spadgett changing to a div is a bigger change it seems like, a lot of styles are wrapped up in it being a pre tag, i'd say lets not mess with it right now

**Comment by spadgett** [comment]
LGTM

**Comment by jwforres** [comment]
[merge]

**Pull request merged**